source/includes/documentation/_vue_js_elements.md

```diff
@@ -4083,72 +4083,95 @@ Available methods in Java JDI Light:
+    <span class="v-chip__content">
+    Green Chip
+  </span>
+  </span>
+</div>
+```
 Available methods in Java JDI Light:
 
-|Method | Description | Return Type
---- | --- | ---
-**is()** | Returns object for work with assertions | ChipAssert
-**getText()** | Gets chip text | String
-**getBoldTextFromCompositeLabel()** | Gets bold text from the chip's composite label | String
-**getRegularTextFromCompositeLabel()** | Gets regular text from the chip's composite label | String
-**getFullTextFromCompositeLabel()** | Gets full text from the chip's composite label | String
-**click()** | Clicks the chip | void
-**close()** | Clicks the chip's close button | void
-**isDraggable()** | Checks if the chip is draggable | boolean
-**isActive()** | Checks if the chip is active | boolean
+|Method | Description                                   | Return Type
+--- |-----------------------------------------------| ---
+**is()** | Returns object for work with assertions       | ChipAssert
+**getText()** | Gets chip text                                | String
+**click()** | Clicks the chip                               | void
+**close()** | Clicks the chip's close button                | void
+**isDraggable()** | Checks if the chip is draggable               | boolean
+**active()** | Checks if the chip is active                  | boolean
 **isFilterIconDisplayed()** | Checks if the chip's filter icon is displayed | boolean
-**isLabel()** | Checks if the chip is marked as label chip | boolean
-**hasIcon()** | Checks if the chip has an icon | boolean
-**hasImage()** | Checks if the chip has an image | boolean
-**hasVisibleBorder()** | Checks if the chip has a visible border | boolean
-**height()** | Gets chip height | String
-**fontSize()** | Gets chip font size | String
-**fontColor()** | Gets chip font color | String
-**backgroundColor()** | Gets chip background color | String
-**borderColor()** | Gets chip border color | String
+**isLabel()** | Checks if the chip is marked as label chip    | boolean
+**hasImage()** | Checks if the chip has an image               | boolean
+**isEnabled()** | Checks if the chip is enabled                 | boolean
+**getContent()** | Get chip content                              | UIElement
+**fontSize()** | Gets chip font size                           | String
+**borderColor()** | Gets chip border color                        | String
+**hasXSmallSize()** | Checks if the chip has x-small size           | boolean
+**hasSmallSize()** | Checks if the chip has small size             | boolean
+**hasDefaultSize()** | Checks if the chip has default size           | boolean
+**hasLargeSize()** | Checks if the chip has large size             | boolean
+**hasXLargeSize()** | Checks if the chip has x-large size           | boolean
+**isRemovable()** | Checks if the chip is revomable               | boolean
+**isPill()** | Checks if the chip is pill                      | boolean
```

Comment:
we agreed to add info about implemented interfaces like 'mplements HasClick, HasColor, HasIcon, HasImage,
        HasTheme, IsOutlined, HasMeasurement'

Reply:
fixed